### PR TITLE
dev

### PR DESCRIPTION
**Pull request by luiztauffer**

### CODE REVIEW

**Review by alejoe91 — APPROVED** [review]

**Comment by alejoe91** [comment]
@luiztauffer anything else on this PR?

**Comment by luiztauffer** [comment]
@alejoe91 just putting here necessary changes as I go finding them

File: src/spikeinterface_pipelines/pipeline.py [hunk 1]
```diff
@@ -10,18 +10,19 @@
...
-# TODO - WIP
 def run_pipeline(
     recording: si.BaseRecording,
-    scratch_folder: Path = Path("./scratch/"),
-    results_folder: Path = Path("./results/"),
-    job_kwargs: JobKwargs = JobKwargs(),
-    preprocessing_params: PreprocessingParams = PreprocessingParams(),
-    spikesorting_params: SpikeSortingParams = SpikeSortingParams(),
-    postprocessing_params: PostprocessingParams = PostprocessingParams(),
+    scratch_folder: Union[Path, str] = Path("./scratch/"),
```
Comment (alejoe91) [inline src/spikeinterface_pipelines/pipeline.py]: with python > 3.9 we can switch to `Path | str` annotation, which is way nicer!

Comment (luiztauffer) [reply src/spikeinterface_pipelines/pipeline.py]: I agree! I got a problem with the spikeinterface sorter images though, which have 3.9 if I'm not mistaken

**Comment by alejoe91** [comment]
In that case `from __future__ import annotations` does the trick!

**Pull request merged**